virtual-organizations/OSG.yaml

```diff
@@ -214,7 +214,7 @@ DataFederations:
         Authorizations:
           - SciTokens:
               Issuer: https://ap22.uc.osg-htc.org:1094/ospool/ap22
-              Base Path: /ospool/ap22
+              Base Path: /ospool/ap22/data,/ospool/uc-shared/project/s
```

Comment:
```suggestion
              Base Path: /ospool/ap22/data,/ospool/uc-shared/projects
```

Reply:
Stealing your valor by force pushing. No sense in adding /more/ commit noise to this repo